procedures/auxtel/observation_procedures/Parabolic_Focus_Determination_Data_Acq.ipynb

```diff
@@ -9,23 +9,42 @@
     "\n",
```

Comment:
<p>So the focus offsets are all measured relative to the slot(s) being empty, which is why we often start with a CWFS with either the filter and/or grating set to empty.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Fixed in your commit. </p><p> </p>